### PR TITLE
Change some stuff of bug_report.md

### PR DESCRIPTION
**Pull request by ArjunSharda**
This will add on to the bug report template for issues, possibly making it better.

### CODE REVIEW

**Comment by ajsharda17** [comment]
@swift-ci Please test

**Comment by ArjunSharda** [comment]
If possible, can you review? @franklinsch

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Environment (please complete the following information):**
+**Software Version(s) & Device(s) Information (please fill out the following information):**
```
Comment (CodaFi) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: "Environment" is correct here, and more concise

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ assignees: ''
 **Describe the bug**
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+**How To Reproduce**
```
Comment (CodaFi) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Perhaps "Steps To Reproduce"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Environment (please complete the following information):**
+**Enviornment (please fill out the following information)**
```
Comment (CodaFi) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Please revert this, it introduces a typo.

Comment (ArjunSharda) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed the typo

**Comment by CodaFi** [comment]
Note to self: Squash-merge this in

@swift-ci smoke test

**Pull request merged**